src/schemas/json/compile-commands.json

```diff
@@ -0,0 +1,38 @@
+{
+  "schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
Can you make this draft v4 since most editors that support schemastore.org only supports that version? It doesn't look like you need to make any other changes, since you're not using any v6-only constructs in the schema itself